### PR TITLE
qmp_event_notification: add event check for ram and disk suspend.

### PR DESCRIPTION
**Pull request by CongLi**
Signed-off-by: Cong Li <coli@redhat.com>

id: 1343300

### CODE REVIEW

**Comment by CongLi** [comment]
13:39:22 INFO | PASS 1-smp_8.2048m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.qmp_event_notification.qmp_suspend.from_guest
13:48:20 INFO | PASS 1-smp_8.2048m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.qmp_event_notification.qmp_disk_suspend.from_guest

File: qemu/tests/cfg/qmp_event_notification.cfg [hunk 1]
```diff
@@ -50,6 +50,18 @@
             no Windows
             event_cmd = hwclock --systohc
             event_check = "RTC_CHANGE"
+        - qmp_suspend:
+            no Windows
+            event_cmd = pm-suspend
+            event_check = "SUSPEND"
```
Comment (xutian) [inline qemu/tests/cfg/qmp_event_notification.cfg]: is these events support on qemu2.3 and qemu1.5 ?

Comment (CongLi) [inline qemu/tests/cfg/qmp_event_notification.cfg]: Yes.
# grep -nr "SUSPEND_DISK" . (same as SUSPEND)
./caps_1.2.2.x86_64.replies:292:            "name": "SUSPEND_DISK"
./caps_1.3.1.x86_64.replies:330:            "name": "SUSPEND_DISK"
./caps_1.4.2.x86_64.replies:342:            "name": "SUSPEND_DISK"
./caps_1.5.3.x86_64.replies:360:            "name": "SUSPEND_DISK"
./caps_1.6.0.x86_64.replies:366:            "name": "SUSPEND_DISK"
./caps_1.6.50.x86_64.replies:375:            "name": "SUSPEND_DISK"
./caps_2.1.1.x86_64.replies:450:      "name": "SUSPEND_DISK"
./caps_2.4.0.x86_64.replies:504:      "name": "SUSPEND_DISK"
./caps_2.5.0.x86_64.replies:504:      "name": "SUSPEND_DISK"
./caps_2.6.0-gicv2.aarch64.replies:492:      "name": "SUSPEND_DISK"
./caps_2.6.0-gicv3.aarch64.replies:492:      "name": "SUSPEND_DISK"
./caps_2.6.0.ppc64le.replies:489:      "name": "SUSPEND_DISK"
./caps_2.6.0.x86_64.replies:504:      "name": "SUSPEND_DISK"

**Comment by CongLi** [comment]
@PyLearner 

Could you help review it again? usb device has been removed.

18:04:47 INFO | PASS 1-smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.RHEL.7.3.x86_64.io-github-autotest-qemu.qmp_event_notification.qmp_disk_suspend.from_guest

Thanks.

**Comment by PyLearner** [comment]
Acked-by: Meng Yang meyang@redhat.com

**Comment by ngu-niny** [comment]
No problem any more, test passed:

11-smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.RHEL.7.3.x86_64.io-github-autotest-qemu.qmp_event_notification.qmp_suspend.from_guest Finshed 2016-07-14 02:04:45 2016-07-14 02:06:41 PASS 116.572535
12-smp_4.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.RHEL.7.3.x86_64.io-github-autotest-qemu.qmp_event_notification.qmp_disk_suspend.from_guest Finshed 2016-07-14 02:06:41 2016-07-14 02:08:44 PASS 123.149022

Acked-by: Nini Gu <ngu@redhat.com>

**Comment by xutian** [comment]
LGTM,

Merged thanks @CongLi , @ngu-niny and @PyLearner

**Pull request merged**